ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -413,6 +413,12 @@ public sealed class ArchiWebHandler : IDisposable {
 					return (false, mobileTradeOfferIDs);
 				}
 
+				if (response.StatusCode.IsServerErrorCode() && !string.IsNullOrEmpty(response.Content.ErrorText)) {
```

Comment:
This response can't have a server error, because request doesn't ask for returning those.